test/test_proxy_tensor.py

```diff
@@ -1090,6 +1090,9 @@ def f(a, b, c, d, e):
     xfail('nanquantile'),
     xfail('narrow'),
 
+    # couldn't find symbolic meta function/decomposition
+    skip('view_copy'),
```

Comment:
Can this be changed to `xfail`? 

We prefer `xfail` as the test will start failing if it is fixed in future. `skip` hides that.

Reply:
This was xfailing all the tests in the file which was causing issues since some tests pass, but it is no longer required after a rebase